projects/packages/backup/src/js/components/Admin/index.js

```diff
@@ -20,26 +20,56 @@ import Backups from '../Backups';
 import BackupStorageSpace from '../backup-storage-space';
 import ReviewRequest from '../review-request';
 import Header from './header';
-import { useIsFullyConnected } from './hooks';
+import {
+	useIsFullyConnected,
+	useIsSecondaryAdminNotConnected,
+	useSiteHasBackupProduct,
+} from './hooks';
 import NoBackupCapabilities from './no-backup-capabilities';
 import './style.scss';
 import '../masthead/masthead-style.scss';
 
 /* eslint react/react-in-jsx-scope: 0 */
 const Admin = () => {
-	const [ connectionStatus ] = useConnection();
+	const [ connectionStatus, , BackupSecondaryAdminConnectionScreen ] = useConnection();
```

Comment:
I think it would be better to make `useConnection()` to return an object instead of an array. It might require some refactoring, so we could address this in another PR.